src/components/menu-icon/menuIcon.tsx

```diff
@@ -0,0 +1,16 @@
+import { h, Component } from "preact";
+import * as styles from "./menuIcon.scss";
+
+export class MenuIcon extends Component {
```

Comment:
since the icon doesn't have any logic, you can return the div directly in the plugin.ts instead of wrapping it